cmd/explaintest/r/partition_pruning.result

```diff
@@ -3859,82 +3859,51 @@ TableReader_8	10.00	root		data:Selection_7
+  └─TableFullScan_6	10000.00	cop[tikv]	table:t2, partition:p2	keep order:false, stats:pseudo
 explain select * from t1 where a >= 0xFE AND a <= 0xFF;
 id	estRows	task	access object	operator info
 TableReader_8	250.00	root		data:Selection_7
 └─Selection_7	250.00	cop[tikv]		ge(test.t1.a, 254), le(test.t1.a, 255)
   └─TableFullScan_6	10000.00	cop[tikv]	table:t1, partition:p2	keep order:false, stats:pseudo
 explain select * from t2 where a >= 0xFE AND a <= 0xFF;
 id	estRows	task	access object	operator info
-PartitionUnion_9	750.00	root		
-├─TableReader_12	250.00	root		data:Selection_11
-│ └─Selection_11	250.00	cop[tikv]		ge(test.t2.a, 254), le(test.t2.a, 255)
-│   └─TableFullScan_10	10000.00	cop[tikv]	table:t2, partition:p0	keep order:false, stats:pseudo
-├─TableReader_15	250.00	root		data:Selection_14
-│ └─Selection_14	250.00	cop[tikv]		ge(test.t2.a, 254), le(test.t2.a, 255)
-│   └─TableFullScan_13	10000.00	cop[tikv]	table:t2, partition:p1	keep order:false, stats:pseudo
-└─TableReader_18	250.00	root		data:Selection_17
-  └─Selection_17	250.00	cop[tikv]		ge(test.t2.a, 254), le(test.t2.a, 255)
-    └─TableFullScan_16	10000.00	cop[tikv]	table:t2, partition:p2	keep order:false, stats:pseudo
+TableReader_8	250.00	root		data:Selection_7
+└─Selection_7	250.00	cop[tikv]		ge(test.t2.a, 254), le(test.t2.a, 255)
+  └─TableFullScan_6	10000.00	cop[tikv]	table:t2, partition:p2	keep order:false, stats:pseudo
 explain select * from t1 where a < 64 AND a >= 63;
 id	estRows	task	access object	operator info
 TableReader_8	250.00	root		data:Selection_7
 └─Selection_7	250.00	cop[tikv]		ge(test.t1.a, 63), lt(test.t1.a, 64)
   └─TableFullScan_6	10000.00	cop[tikv]	table:t1, partition:p0	keep order:false, stats:pseudo
 explain select * from t2 where a < 64 AND a >= 63;
 id	estRows	task	access object	operator info
-PartitionUnion_9	750.00	root		
-├─TableReader_12	250.00	root		data:Selection_11
-│ └─Selection_11	250.00	cop[tikv]		ge(test.t2.a, 63), lt(test.t2.a, 64)
-│   └─TableFullScan_10	10000.00	cop[tikv]	table:t2, partition:p0	keep order:false, stats:pseudo
-├─TableReader_15	250.00	root		data:Selection_14
-│ └─Selection_14	250.00	cop[tikv]		ge(test.t2.a, 63), lt(test.t2.a, 64)
-│   └─TableFullScan_13	10000.00	cop[tikv]	table:t2, partition:p1	keep order:false, stats:pseudo
-└─TableReader_18	250.00	root		data:Selection_17
-  └─Selection_17	250.00	cop[tikv]		ge(test.t2.a, 63), lt(test.t2.a, 64)
-    └─TableFullScan_16	10000.00	cop[tikv]	table:t2, partition:p2	keep order:false, stats:pseudo
+PartitionUnion_8	500.00	root		
+├─TableReader_11	250.00	root		data:Selection_10
+│ └─Selection_10	250.00	cop[tikv]		ge(test.t2.a, 63), lt(test.t2.a, 64)
+│   └─TableFullScan_9	10000.00	cop[tikv]	table:t2, partition:p0	keep order:false, stats:pseudo
+└─TableReader_14	250.00	root		data:Selection_13
+  └─Selection_13	250.00	cop[tikv]		ge(test.t2.a, 63), lt(test.t2.a, 64)
+    └─TableFullScan_12	10000.00	cop[tikv]	table:t2, partition:p1	keep order:false, stats:pseudo
```

Comment:
This branch does not handle `>=` `<=` like the master branch, so it an extra partition.
That's OK.